src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java

```diff
@@ -245,4 +246,20 @@ else if (PatternUtils.ALPHANUMERIC.matcher(texture).matches()) {
         }
     }
 
+    /**
+     * Adds additional lores for the {@link ItemStack}.
+     *
+     * @param newLore
+     *                New lores to be added to the {@link ItemStack}
+     */
+    public void addLore(String... newLore) {
+        ItemMeta meta = getItemMeta();
+        if (meta == null) return;
```

Comment:
ItemMeta cannot be null.

Reply:
I'm almost sure it could be in 1.13. Or it's my bad memory